components/app-triplestore/src/main/java/org/trellisldp/app/triplestore/AppUtils.java

```diff
@@ -125,6 +133,18 @@ public static RDFConnection getRDFConnection(final TrellisConfiguration config)
         return connect(createTxnMem());
     }
 
+    private static HttpClient buildHttpClient(final TrellisConfiguration config) {
+        final Optional<String> userName = ofNullable(config.getResources().getUserName());
```

Comment:
Building an `Optional` here is unidiomatic. Let `getUserName()` return an `Optional`.